### ISSUE TITLE
unable to make trees

### ISSUE THREAD

**Issue by aamirrr-m** [DessimozLab/fold_tree#15]
**I have a group of custom structures and I attempted to run the workflow and I got hit with this error:** 

_Error in rule foldseek_allvall_1:
    jobid: 8
    input: ./results/finalset.csv
    output: ./results/allvall_1.csv
    log: ./results/logs/foldseekallvall.log (check log file(s) for error details)
    conda-env: foldtree
shell:
/home/aamir/fold_tree/foldseek/foldseek easy-search ./results/structs/ ./results/structs/ ./results/allvall_1.csv ./results/tmp --format-output 'query,target,fident,alnlen,mismatch,gapopen,qstart,qend,tstart,tend,evalue,bits,lddt,lddtfull,alntmscore' --exhaustive-search --alignment-type 2 -e inf --threads 1
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
Complete log: .snakemake/log/2024-01-30T171533.541710.snakemake.log_

**The same thing happened when I changed it and just used an identifiers.txt file:**

_Error in rule foldseek_allvall_1:
    jobid: 8
    input: ./results/finalset.csv
    output: ./results/allvall_1.csv
    log: ./results/logs/foldseekallvall.log (check log file(s) for error details)
    conda-env: foldtree
    shell:
        /home/aamir/fold_tree/foldseek/foldseek easy-search ./results/structs/ ./results/structs/ ./results/allvall_1.csv ./results/tmp --format-output 'query,target,fident,alnlen,mismatch,gapopen,qstart,qend,tstart,tend,evalue,bits,lddt,lddtfull,alntmscore' --exhaustive-search --alignment-type 2 -e inf --threads 1
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Finished job 1.
3 of 15 steps (20%) done
Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
Complete log: .snakemake/log/2024-01-30T171446.885331.snakemake.log

The foldseekallvall.log file is empty and so I dont know how this problem came about, I have made trees before today and it was fine. I reinstalled it and nothing has changed either._

Thank you for your help in advance.

**Comment by aamirrr-m** [comment]
Ok so I seem to have solved this issue for local installations of foldtree! If you head to fold_tree/workflow/config/config_vars.yaml 

Find the line: 
**foldseek_path: provided**

change it to:
**foldseek_path: foldseek**

This fixed the issue for me, seems as though the provided foldseek given in the respository is out of date or has another issue

**Comment by cactuskid** [comment]
hey. Thanks for catching this! Yes, I had configured things to use an old version of foldseek. It's the one I had used when I got my benchmarking results since it was still under very active development at the time and I wanted my results to be replicable. For normal use cases this should not impact anything. I changed this parameter in the configfile on the repo so it should automatically change this for the collab implementation too.
